src/current/v24.3/troubleshoot-replication-zones.md

```diff
@@ -0,0 +1,219 @@
+~~~
+
+<a name="step1"></a>
+
+### Step 1. Start with a target schema object
+
+Start by looking at the zone configuration for the schema object that you think may be misconfigured. Depending on the [type of problem](#types-of-problems), one way you may have come to this conclusion is by [monitoring the critical nodes endpoint](#monitoring).
+
+Use the [`SHOW ZONE CONFIGURATION`]({% link {{ page.version.version }}/show-zone-configurations.md %}) statement to inspect the target object's zone configurations.
+
+For example, this is the zone configuration for the `movr.rides` table:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW ZONE CONFIGURATION FOR TABLE movr.users;
+~~~
+
+~~~
+----------------+-------------------------------------------------------------------------------------------
+  DATABASE movr | ALTER DATABASE movr CONFIGURE ZONE USING
+                |     range_min_bytes = 134217728,
+                |     range_max_bytes = 536870912,
+                |     gc.ttlseconds = 14400,
+                |     num_replicas = 5,
+                |     num_voters = 3,
+                |     constraints = '{+region=europe-west1: 1, +region=us-east1: 1, +region=us-west1: 1}',
+                |     voter_constraints = '[+region=us-east1]',
+                |     lease_preferences = '[[+region=us-east1]]'
+(1 row)
+~~~
+
+In this case, the output is expected for a multi-region cluster spread across 9 nodes that was configured using [Multi-region SQL]({% link {{ page.version.version }}/multiregion-overview.md %}). Since nothing has been manually modified, the output shows that `movr.users` is using the zone configuration from its parent `movr` database. For more information about how this works, see [How zone config inheritance works]({% link {{ page.version.version }}/configure-replication-zones.md %}#how-zone-config-inheritance-works).
+
+However, if the zone configuration had been manually modified, there could be inconsistencies in the output that would show a misconfiguration.
+
+For example:
+
+- If the [type of problem](#types-of-problems) were a [constraint violation](#monitoring), you'd want to check that the values in [`constraints`]({% link {{ page.version.version }}/configure-replication-zones.md %}#constraints), [`voter_constraints`]({% link {{ page.version.version }}/configure-replication-zones.md %}#voter_constraints), and [`lease_preferences`]({% link {{ page.version.version }}/configure-replication-zones.md %}#lease_preferences) were not logically inconsistent, which would cause constraint violations.
+- If the [type of problem](#types-of-problems) were an [under-replicated range](#monitoring), you'd want to check the values of [`num_replicas`]({% link {{ page.version.version }}/configure-replication-zones.md %}#num_replicas) and [`num_voters`]({% link {{ page.version.version }}/configure-replication-zones.md %}#num_voters). Modifying these values can cause under-replication.
```

Comment:
Is it possible to clarify that (I presume) this means one or both of those values would be set too low?

Reply:
For sure! Added the following, which is somewhat taken from our existing non-voting replicas docs, but modified a little 

> For example, `num_replicas` could be set too low. It's also easy to make an arithmetic mistake when configuring `num_voters`; if `num_voters` is less than `num_replicas`, the difference dictates the number of [non-voting replicas]({% link {{ page.version.version }}/architecture/replication-layer.md %}#non-voting-replicas). This is why most users should control non-voting replica placement with the high-level [multi-region SQL features]({% link {{ page.version.version }}/multiregion-overview.md %}) instead.